### DISCUSSION TITLE
Getting My Own Transactions

### DISCUSSION THREAD

**Discussion by raycekar** [mmohades/Venmo#60] [Q&A] [answered]
Is there a way to get my own transactions?  Or, is there a way to get my user id so that I can get my transactions?

**Comment by mmohades** [comment, marked as answer]
Yes `client.my_profile()` would return your own User object.
https://github.com/mmohades/Venmo/blob/master/venmo_api/venmo.py#L19
You may then get your id to get your transactions.
